src/views/Dashboard.vue

```diff
@@ -267,14 +267,15 @@ export default {
   },
   methods: {
     onToggleSearchbar() {
-      if (this.searchbarOpen) {
-        this.$refs.searchbar.blurInput()
-        this.searchbarOpen = !this.searchbarOpen
-        this.$store.dispatch('filters/updateSearchQuery', '')
-      } else {
-        this.searchbarOpen = !this.searchbarOpen
-        this.$refs.searchbar.focusInput()
-      }
+      this.searchbarOpen = !this.searchbarOpen
```

Comment:
@nigina23 können wir, um die Lesbarkeit zu erhöhen, das umschalten des Status hinter die Condition innerhalb des $nextTick setzen?

So wird searchbarOpen im aktuellen State vollständig verwendet und erst danach und im selben Tick wird der State geändert.

`
this.$nextTick =>..
  if () ....
  this.searchbarOpen = !this....
})
`

Reply:
@nigina23 Ich habe verstanden, wieso du das so rum gebaut hast. Es scheint nicht anders möglich zu sein. Gute Arbeit!